dev-client/src/util.ts

```diff
@@ -69,3 +69,15 @@ export const removeKeys = (a: any, b: any) => {
     }
   }
 };
+
+export const normalizeText = (text: string) =>
```

Comment:
issue: this duplicates code in the web client. Please put it in shared client and update both web and mobile client to use that.

Reply:
I made a separate issue for this: https://github.com/techmatters/terraso-mobile-client/issues/643 . This code isn't new in this PR just moved to a different file, and I would rather tackle that in another PR.